src/earner.py

```diff
@@ -210,13 +214,23 @@ def __process_earn(
                 )
         except Exception as e:
             self.logger.error(f"Error processing earn tx: {e}")
-            send_error_to_discord(
-                sett_name,
-                "Earn",
-                error=e,
-                chain=self.chain,
-                keeper_address=self.keeper_address,
-            )
+            if vault and vault.address in CRITICAL_VAULTS:
+                send_error_to_discord(
```

Comment:
we'll want to alert a specific role when this alert fires so i dont think we can reuse this funciton. honestly we should rework the whole discord messaging arch of this because it's really rigid. different issue though

Reply:
Done